pkg/virt-operator/kubevirt.go

```diff
@@ -511,6 +511,36 @@ func NewKubeVirtController(
 		return nil, err
 	}
 
+	_, err = c.informers.ClusterInstancetype.AddEventHandler(cache.ResourceEventHandlerFuncs{
```

Comment:
We should:
```
	cache.WaitForCacheSync(stopCh, c.informers.ClusterInstancetype.HasSynced)
	cache.WaitForCacheSync(stopCh, c.informers.ClusterPreference.HasSynced)
```

Reply:
I'm not sure it's possible because initially these CRDs won't be available (deployed by virt-operator) and the operator would therefore never start. They can only sync after the CRDs are installed on the cluster.

Reply:
I think this is the same for other CRDs as Felix says.